src/app/profile/overview/overview.component.less

```diff
@@ -62,4 +72,41 @@
 
 .wrap { word-wrap: break-word; }
 
-.profile-update-button { white-space: normal; }
+.profile-update-button {
+  color: #d3d3d3;
```

Comment:
The color should be white, per PatternFly spec - you shouldn't need to set the font color.

Reply:
@mindreeper2420 I've set the font color because as per design the color is not pure white that comes with standard pf button.